Frontend/ui-driver/src/Flow.purs

```diff
@@ -595,6 +599,33 @@ handleDeepLinksFlow event activeRideResp isActiveRide = do
     _ -> pure unit
   checkPreRequisites activeRideResp isActiveRide
 
+gullakDeeplinkFlow :: String -> FlowBT String Unit
+gullakDeeplinkFlow event = do
+  let token = getValueToLocalStore GULLAK_TOKEN
+      gullakRemoteConfig = CommonRC.gullakConfig $ getValueToLocalStore DRIVER_LOCATION
+      splitLinkArr = split (Pattern "+") event
+      loadDynamicModule = fromMaybe false $ UC.runFn3 DU.getAnyFromWindow "loadDynamicModule" Nothing Just
+      showVideo = (fromMaybe "" (splitLinkArr !! 1)) == "true"
+      installOnly = (fromMaybe "" (splitLinkArr !! 2)) == "true"
+      videoLink = fromMaybe "" gullakRemoteConfig.videoUrl
+  if not loadDynamicModule then homeScreenFlow else do
+    push <- lift $ lift $ liftFlow $ getPushFn Nothing "BenefitsScreen"
+    if HU.isTokenWithExpValid token then do
+      void $ pure $ UC.runFn4 JB.emitJOSEventWithCb "gl_sdk" (JB.josEventInnerPayload {param1 = token, param2 = show installOnly}) push BSC.GullakSDKResponse
+      pure unit
+    else do
+      response <- lift $ lift $ HelpersAPI.callApi $ API.GetSdkTokenReq "0" API.Gullak
+      case response of
+        Left _ -> do
+          void $ pure $ JB.toast $ getString ERROR_OCCURED_PLEASE_TRY_AGAIN_LATER
+          pure unit
+        Right (API.GetSdkTokenResp resp) -> do
+          void $ pure $ setValueToLocalStore GULLAK_TOKEN $ resp.token <> "<$>" <> fromMaybe "" resp.expiry
```

Comment:
It is better to use a single character to distinguish strings.

Reply:
@spravinkumar9952  It's already there in this format, If we change It won't be backward compatible